preview/Win7Msix/Win7MSIXInstaller/MsixRequest.hpp

```diff
@@ -62,6 +66,9 @@ class MsixRequest
     /// @return can return null if called before CreateAndShowUI or if Flags::QuietUX was passed in and there is no UI.
     UI* GetUI() { return m_UI; }
     bool IsQuietUX() { return (m_flags & Flags::QuietUX) == Flags::QuietUX; }
+
+    /// Prepares and sends msixrequest to uninstall the package in case cancel button was clicked during installation
+    HRESULT UnInstallPackage(std::wstring packageFullName);

```

Comment:
>UnInstallPackage [](start = 12, length = 16)

If we want a first class notion of cancel we have to figure out a different way to expose it. It should not be an API on the request.

Reply:
Modified and moved to error handler as per design discussion.